aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -567,6 +576,16 @@ Tensor randint_like(
   return result.random_(0, high, c10::nullopt);
```

Comment:
Make is similar to `randint` and `randint_out` above: reuse new `randint_like` here
```suggestion
  return native::randint_like(self, high, c10::nullopt, options, optional_memory_format);
```